.github/workflows/build_and_push_sdk_packages.yaml

```diff
@@ -0,0 +1,32 @@
+name: SDK Publishing
+
+on:
+  workflow_run:
+    workflows: ["SDK Packaging"]
+    types: [completed]
+    branches-ignore:
```

Comment:
Do we need to ignore main if we're running on tags below?

Reply:
I wanted to avoid main until the procedure is well tested, since we cannot push more than once to test/pypi (maybe we can manually delete versions, though). But of course, this procedure should be run on main, and we should only use semver tags on main (unless we'd be juggling more than one minor or major version on support at once, but I don't think we'll take that route soon or at all).

Reply:
We will trigger the release workflow only from main and based on semver tags.